Code/Framework/AzCore/Tests/Asset/AssetManagerLoadingTests.cpp

```diff
@@ -1863,6 +1848,7 @@ namespace UnitTest
     TEST_F(AssetJobsFloodTest, ContainerLoadTest_SimpleCircularPreload_LoadsRoot)
 #endif // !AZ_TRAIT_DISABLE_FAILED_ASSET_MANAGER_TESTS
     {
+        m_traceBusDisableToken = {};
```

Comment:
If you want to make this read easier, a function could be added to Code/Framework/AzCore/Tests/Asset/BaseAssetManagerTest.cpp like "EnableTraceBus()" to make the intent more straight forwards here. Just a thought.